src/main/kotlin/frc/chargers/hardware/sensors/DriverCamManager.kt

```diff
@@ -0,0 +1,27 @@
+package frc.chargers.hardware.sensors
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard
+import edu.wpi.first.networktables.NetworkTable
+import edu.wpi.first.networktables.NetworkTableEntry
+import edu.wpi.first.networktables.NetworkTableInstance
+import edu.wpi.first.cameraserver.CameraServer
+
+public class DriverCamManager(
+    public val totalDriverCams: Int = 0,
+    public val defaultResWidth: Int = 720,
+    public val defaultResHeight: Int = 1280){
```

Comment:
I'm not sure we want to have default parameters here unless there's a particularly compelling reason—does WPILib use 720x1280 by default or is that just the resolution of the cameras we happen to have?

Reply:
you're right, I should probably change that.